tests/core/test_linalg.py

```diff
@@ -125,6 +125,19 @@ def test_inv(app_inst: ArrayApplication):
         assert np.allclose(np.linalg.cholesky(mat.get()), L, rtol=1e-4, atol=1e-4)
 
 
+def test_inv_uppertri(app_inst: ArrayApplication):
+    block_shape, shape = (3, 3), (20, 20)
+    R_np = np.arange(1, shape[0] ** 2 + 1, 1).reshape(*shape).astype(float)
+    R_np = np.triu(R_np)
+
+    R = app_inst.array(R_np, block_shape=block_shape)
+
+    R_inv_naive = linalg.inv(app_inst, R).get()
```

Comment:
I was wondering if we should explicitly test here with ground truth rather than testing it against it's sibling

Reply:
@vinamrabenara Please tell me if you feel strongly about the explicit testing, but I believe the transitive law is at play here. In a previous test, we test that `linalg.inv == np.linalg.inv` with a tolerance of `1e-4`, so then intuitively it must mean that on line `135`, we are essentially using the explicit ground truth, but using a different API.